src/Microsoft.Data.SqlClient/NuGet.config

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!--
+      During development, the Abstractions package that MDS depends on may only
+      be available locally.
+
+      This directory must exist before MDS restore will succeed.  Building and
+      packing the Abstractions package will create it.
+    -->
+    <add key="local" value="../../packages/" />
```

Comment:
If you're adding this make sure to add an empty packages folder and put a .gitkeep file in there. Otherwise, as you noted restore fails.

However, I'm a bit skeptical of forcing this into the repo. Imo, adding more nuget.config files complicates the tree and can cause difficult to diagnose/silent errors during restore. This seems to me like something a developer should be able to do on their own in the handful of instances where its necessary, but not something that needs to be baked into the repo.